client/spa/js/learning-resource/spec/learning-resource.view.spec.js

```diff
@@ -0,0 +1,145 @@
+describe('Learning resource view ', function(){
+
+  var model;
+  var view;
+
+  beforeEach(function(){
+    var Model = Backbone.Model.extend({});
+
+    // Add some convenience tests for working with the DOM
+    // Helper for toHaveClass
+    jasmine.addMatchers(matchers);
+
+    // Needs to have the fields required by the template
+    model = new Model({
+      title: 'Crisis Averted',
+      resourceType: 'presentation',
+      description: 'You are welcome.',
+      authors: ['Mary', 'Joe']
+    });
+
+    view = new View({
+      model: model
+    });
+  });
+
+  describe('when the view is instantiated ', function(){
+
+    it('creates the correct element', function(){
+      // Element has to be uppercase
+      expect(view.el.nodeName).toEqual('DIV');
+    });
+
+    it('sets the correct class', function(){
+      expect(view.$el).toHaveClass('learning-resource');
+    });
+
+  });
+
+  describe('when the view is rendered ', function(){
+
+    it('returns the view object ', function(){
+      expect(view.render()).toEqual(view);
+    });
+
+    it('produces the correct HTML ', function(){
+      view.render();
+      expect(view.$('#title').val()).toEqual('Crisis Averted');
+    });
+
+  });
+
+  describe('when the user clicks on the Edit button ', function(){
+
+    beforeEach(function(){
+      view.$('button .b-edit').trigger('click');
+      view.render();
+    });
+
+    describe('and when the user inputs new information ', function(){
+
+      describe('then clicks the cancel button', function(){
+
+        beforeEach(function(){
+          view.$('#title').val('changed title');
+          view.$('#desc').val('changed description');
+          view.$('#authors').val('sis');
+          view.$('#resourceType option:selected').val('link');
+          view.render();
+        });
+
+        it('cancels the user input', function(){
+          view.$('button .b-cancel').trigger('click');
+        });
+
+        it('should see the initial information', function(){
+          view.render();
+          expect(view.$('#title').val()).toEqual('Crisis Averted');
+          expect(view.$('#desc').val()).toEqual('You are welcome.');
+          expect(view.$('#auth').val()).toContain('Joe');
+          expect(view.$('#resourceType option:selected').val()).toEqual('presentation');
+        });
+      });
+
+      describe('and then clicks on the Update button ', function(){
+
+        beforeEach(function(){
+          view.$('#title').val('changed title');
+          view.$('#desc').val('changed description');
+          view.$('#authors').val('sis');
+          view.$('#resourceType option:selected').val('link');
+        });
+
+        it('updates the model', function(){
+          view.$('button .b-update').trigger('click');
+          view.render();
+        });
+
+        it('should see the changed information', function(){
+          view.render();
+          expect(view.$('#title').val()).toEqual('changed title');
```

Comment:
By checking the field value with $ you are really testing jquery instead of your code. You want to test that your save function was called or that the model was updated properly.